CHANGELOG.md

```diff
@@ -4,7 +4,12 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 ## Next
 
+### Added
+
+- Add Mint up by @mollyIV
```

Comment:
Would you mind adding a link to the PR? That will make it easier in the future to check out the PR that implemented this.